src/view/ModuleCompatibilityCheckerShell.svelte

```diff
@@ -0,0 +1,47 @@
+<svelte:options accessors={true} />
+
+<script>
+	/**
+	 * This is the main `ApplicationShell` component. Await upon spreadsheet fetch and either show the table or an
+	 * error. Also coordinates debouncing / serializing app state / position to session storage on positional changes.
+	 */
+	import { getContext, setContext } from "svelte";
+
+	import { ApplicationShell } from "@typhonjs-fvtt/runtime/svelte/component/core";
+	import { debounce } from "@typhonjs-fvtt/runtime/svelte/util";
+
+	import { spreadsheetStore } from "../store/SpreadsheetStore.js";
```

Comment:
There's a typo here: `spreadsheetStore.js` is not capitalized

Reply:
Fixed.. I'll update the PR...